pallets/ah-migrator/src/account.rs

```diff
@@ -135,7 +135,7 @@ impl<T: Config> Pallet<T> {
 			);
 		}
 
-		log::debug!(
+		log::trace!(
```

Comment:
not sure if we need to store these logs, might be too much. or would need to at least test it